README.md

```diff
@@ -1,73 +1,86 @@
 # IRIDA StarAMR Results
 
-IRIDA StarAMR Results program enables StarAMR analysis results that were run through IRIDA to be downloaded into a spreadsheet using the command line. 
+IRIDA StarAMR Results program enables StarAMR analysis results of a single project that were run through IRIDA to be batch downloaded into a single spreadsheet using command line. An option to split the results into separate spreadsheets is also available.
 
-## Running from source code
+# How to use:
 
+- Assuming you have already installed the program, you can use `irida-staramr-results` command.
+
+   ### Examples
+   ```
+   $ irida-staramr-results --help
+   ```
+   ```
+   $ irida-staramr-results -u admin -pw password1 -c /path/to/conf.yml -p 1 -o out -fd 2021-01-02 -td 2021-04-31
+   ```
+- There a few arguments it accepts:
+
+   ## Arguments
+
+   #### Required:
+
+   | Name | Shortcut | Type | Example | Description |
+   |------|----------|------|---------|-------------|
+   |`--username`|`-u`| `string` | admin |This is your IRIDA account username.|
+   |`--password`|`-pw`| `string` | password1 |This is your IRIDA account password.|
+   |`--project`|`-p`| `int` | 1 |Project(s) to scan for StarAMR results.|
+   |`--output`|`-o`| `string` | out |The name of the output excel file.|
+   |`--config`|`-c`| `string` | /path/to/conf.yml |Path to a configuration file. [See configuration details here.](#Configuration-for-IRIDA-REST-API)|
+
+   #### Optional:
+
+   | Name | Shortcut | Type | Example | Description |
+   |------|----------|------|---------|-------------|
+   |`--help`|`-h`|N/A|N/A|Show help message.|
+   |`--version`|`-v`|N/A|N/A|The current version of irida-staramr-results.|
+   |`--separate_all`|`-sa`|N/A|N/A|Export each analysis results into separate output files resulting to one `.xlsx` file per analysis.|
+   |`--from_date`|`-fd`|`string`|2021-01-03|Download only results of the analysis that were created **from** this date.*|
+   |`--to_date`|`-td`|`string`|2021-04-01|Download only results of the analysis that were created **to** this date.*|
+
+   __Notes:__ 
+   - \* Dates are formatted as `YYYY-mm-dd` (eg. 2021-04-08) and include hours from 00:00:00 to 23:59:59 of the inputted date.
+
+# Setup
 ### Python
-IRIDA StarAMR Results requires Python version 3.8 or later. Check the Python version you are using with:
-```
-$ python --version
-```
+   IRIDA StarAMR Results requires **Python version 3.8 or later**. Check the Python version you are using with:
+   ```
+   $ python --version
+   ```
+
+### Configuration for IRIDA REST API
+- You will need to create a client in IRIDA. Refer to [IRIDA client configuration](https://irida.corefacility.ca/documentation/user/administrator/#managing-system-clients) guide for more details.
```

Comment:
Add a note here that this is for administrators, and that users should contact their IRIDA admin for client credentials.

Reply:
done